met-web/src/components/layout/Header/PublicHeader.tsx

```diff
@@ -57,7 +57,12 @@ const PublicHeader = () => {
                         {headerTitle}
                     </Header1>
                     <When condition={isLoggedIn}>
-                        <Button color="inherit" onClick={() => UserService.doLogout()}>
+                        <Button
+                            role="button"
```

Comment:
This element already renders as a <button>, but it doesn't really matter because this `When` condition will never be true while a user is being shown the public header.